extensions/commands/deploy-lipo/xcode/example.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,1020 @@
+// !$*UTF8*$!
```

Comment:
Maybe the xcode project is not necessary, just running the deployer and validating the final library is perfectly good too cover it with testing.